src/main/kotlin/org/jitsi/nlj/transform/node/incoming/RtcpTermination.kt

```diff
@@ -69,7 +70,7 @@ class RtcpTermination(
         }
         //TODO: keep an eye on if anything in here takes a while it could slow the packet pipeline down
         packetReceiveCounts.merge(packetInfo.packet::class.simpleName!!, 1, Int::plus)
-        rtcpEventNotifier.notifyRtcpReceived(packetInfo)
+        rtcpEventNotifier.notifyRtcpReceived(packetInfo.packetAs<RtcpPacket>(), packetInfo.receivedTime)
```

Comment:
Why add the `receivedTime` param to methods if it is part of `PacketInfo`?

Reply:
I see it's different for some of the compound packets, but couldn't we clone the receive timesamp in the RtcpPacket instance?

Reply:
it's because i need to be able to pass an individual rtcp packet, but the packetInfo contains the entire compound rtcp packet.  so instead of changing the offset of the packet in packetInfo (which would be a pain because i'd have to keep track of the original, etc.) i just pass an individual packet (which i get from iterating over the compound packet) and the received time (which is the same for all of them)